Twig/Extension/DataGridExtension.php

```diff
@@ -321,38 +257,28 @@ public function datagridColumnActionCellActionWidget(CellViewInterface $view, $a
         return $this->renderTheme($dataGridView, $context, $blockNames);
     }
 
-    /**
-     * Render html element attributes.
-     * This function is only for internal use.
-     *
-     * @param array $attributes
-     * @param null $translationDomain
-     * @return string
-     */
-    public function datagridAttributes(array $attributes, $translationDomain = null)
+    public function datagridAttributes(array $attributes, ?string $translationDomain = null): string
     {
         $attrs = [];
 
         foreach ($attributes as $attributeName => $attributeValue) {
-            if ($attributeName == 'title') {
-                $attrs[] = $attributeName . '="' . $this->environment->getExtension('translator')->trans($attributeValue, [], $translationDomain) . '"';
-                continue;
+            if ($attributeName === 'title') {
+                $attributeValue = $this->environment
```

Comment:
If we were to inject the `translator` instead of pulling it out of the environment, would that not be better?